recipes-kernel/linux/linux-rockchip_5.10/0021-arm64-dts-rockchip-add-uio-support-for-header-gpios.patch

```diff
@@ -0,0 +1,62 @@
+From b7a44b2d8d3ee51a579d491d83a7faaca076088f Mon Sep 17 00:00:00 2001
+From: Christian Biermann <bie@iesy.com>
+Date: Wed, 25 Oct 2023 15:55:30 +0200
+Subject: [PATCH 1/1] arm64: dts: rockchip: add uio support for header gpios
+
+necessary to activate the pullups of the gpio pins
+---
+ .../boot/dts/iesy/iesy-rpx30-eva-mi-v2.dts      | 17 +++++++++++++++--
+ 1 file changed, 15 insertions(+), 2 deletions(-)
+
+diff --git a/arch/arm64/boot/dts/iesy/iesy-rpx30-eva-mi-v2.dts b/arch/arm64/boot/dts/iesy/iesy-rpx30-eva-mi-v2.dts
+index 0cf0cedb5362..d1ed19e2f84e 100644
+--- a/arch/arm64/boot/dts/iesy/iesy-rpx30-eva-mi-v2.dts
++++ b/arch/arm64/boot/dts/iesy/iesy-rpx30-eva-mi-v2.dts
+@@ -21,6 +21,10 @@ / {
+ 	model = "iesy RPX30 EVA-MI V2 (Eval Kit)";
+ 	compatible = "rockchip,px30-evb-ddr3-v10-linux", "rockchip,px30";
+ 
++	chosen {
++		bootargs = "earlycon=uart8250,mmio32,0xff160000 console=ttyFIQ0 rw root=PARTUUID=614e0000-0000 rootwait uio_pdrv_genirq.of_id=generic-uio";
```

Comment:
I'm not quite sure if setting root to a fixed PARTUUID would allow this to boot on a different module

Reply:
I copied the bootargs from the included rk3326-linux.dtsi and only extented the uio argument.
This should work as well as before.